examples/read_log.py

```diff
@@ -0,0 +1,53 @@
+from pyspecdata import *
+from pyspecProcScripts import logobj
+import h5py
+from matplotlib.pyplot import subplots
+from numpy import log10
+
+coupler_atten = 22
+myfilename = search_filename(
+    "230626_batch230515_E37_Ras_B10_ODNP_1.h5",
+    exp_type="ODNP_NMR_comp/ODNP",
+    unique=True,
+)
+with figlist_var() as fl:
+    # {{{ open h5 file to real log
+    with h5py.File(myfilename, "r") as f:
+        log_grp = f["log"]
+        thislog = logobj.from_group(log_grp)
+        read_array = thislog.total_log
+        read_dict = thislog.log_dict
+    # }}}
+    # {{{ separate the time, RX, power, and commands from the
+    #     log array
+    for j in range(len(read_array)):
+        thistime, thisrx, thispower, thiscmd = read_array[j]
```

Comment:
What is this? This loop isn't doing anything?

Reply:
didn't realize that none of these are used, my bad should have realized before asking you to review!